src/types/lineChart.ts

```diff
@@ -1,16 +1,14 @@
 // SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
 // SPDX-License-Identifier: Apache-2.0
 
-import {Color} from '@amcharts/amcharts5/.internal/core/util/Color';
-
 /**
  * Represents the data for a line chart.
  */
 export interface LineChartData {
   /**
    * The values for the line chart.
    */
-  values: {day: string; value: number | number[]}[];
+  values: {day: string; value: number | number[]; openValue?: number | number[]}[];
```

Comment:
In the linechart component this always assumes values is `number`...
Do we have a case for `number[]` anywhere or can this go?

```suggestion
  values: {day: string; value: number; openValue?: number}[];
```